workshop.pl

```diff
@@ -849,11 +850,33 @@ sub arg_handler {
 } elsif ($userenv_json->{'userenv'}{'properties'}{'packages'}{'manager'} eq "yum") {
     $update_cmd = "yum update --assumeyes";
     $clean_cmd = "yum clean all";
+} elsif ($userenv_json->{'userenv'}{'properties'}{'packages'}{'manager'} eq "apt") {
+    $getsrc_cmd = "apt-get update -y";
+    $update_cmd = "apt-get upgrade -y";
```

Comment:
I usually use `dist-upgrade` instead of `upgrade` as that tends to better handle getting package updates installed.  From the man page:

```
       upgrade
           upgrade is used to install the newest versions of all packages currently installed on the system from the sources enumerated in /etc/apt/sources.list. Packages currently installed with new
           versions available are retrieved and upgraded; under no circumstances are currently installed packages removed, or packages not already installed retrieved and installed. New versions of
           currently installed packages that cannot be upgraded without changing the install status of another package will be left at their current version. An update must be performed first so that
           apt-get knows that new versions of packages are available.

       dist-upgrade
           dist-upgrade in addition to performing the function of upgrade, also intelligently handles changing dependencies with new versions of packages; apt-get has a "smart" conflict resolution system,
           and it will attempt to upgrade the most important packages at the expense of less important ones if necessary. The dist-upgrade command may therefore remove some packages. The
           /etc/apt/sources.list file contains a list of locations from which to retrieve desired package files. See also apt_preferences(5) for a mechanism for overriding the general settings for
           individual packages.
```